APE9.rst

```diff
@@ -0,0 +1,142 @@
+A General Timeseries Class for Astropy
+-------------------------------------
+
+author: Stuart Mumford
+
+date-created: 2016 March 24
+
+date-last-revised: 2017 November 10
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+The goal of a timeseries object in astropy is to provide a core set of
+functionality for representing data where time is the physical event list. This
+includes both continuous values binned in time or event list type data.
```

Comment:
What does "continuous values binned in time mean"? That the timestamps are a time range instead of an individual time?